addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -20,6 +20,29 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         return res;
     },
 
+    /**
+     * @override
+     */
+     getValue: function () {
+        let editable = this._super(...arguments);
+        const parsedDocument = new DOMParser().parseFromString(editable, "text/html");
+        const dropableZone = parsedDocument.querySelector('.o_editable');
+
+        return dropableZone ? dropableZone.innerHTML : editable;
+    },
+
+    /**
+     * @override
+     */
+     setValue: function (value) {
+        const parsedDocument = new DOMParser().parseFromString(value, "text/html");
+        if (this.$editable[0].querySelector('.o_editable') && !parsedDocument.querySelector('.o_editable')) {
```

Comment:
I think looking for drop zones here as well rather than the `o_editable` class would make this condition more easy to understand.

Reply:
@dmo-odoo  this code seems finding this `<div class="col o_mail_no_options o_mail_wrapper_td bg-white oe_structure o_editable" data-editor-message="DRAG BUILDING BLOCKS HERE"><div>` so should we add a class like `o_dropzone` and search it for better understanding?

Reply:
No, you're right in your other comment reply. It's ok to target the `o_editable` class. I didn't realize it was actually _on_ the dropzone itself so it's fine. Thanks.